src/makeRobot.js

```diff
@@ -37,8 +37,59 @@
  *
  * @return {object}
  */
+
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    'name': name,
+    'chip version': version,
+    'wheels': wheels,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+    get info() {
+      return `name: ${this.name}, \
+chip version: ${version}, \
+wheels: ${this.wheels}`;
+    },
```

Comment:
```suggestion
      return `name: ${this.name}, `
      + `chip version: ${version}, `
      + `wheels: ${this.wheels}`;
    },
```
Такий запис виглядає набагато читабельнішим